packages/integration_test/android/build.gradle

```diff
@@ -51,13 +51,12 @@ android {
     dependencies {
         // TODO(egarciad): These dependencies should not be added to release builds.
         // https://github.com/flutter/flutter/issues/56591
-        testImplementation 'junit:junit:4.12'
-        testImplementation 'org.mockito:mockito-inline:5.0.0'
+        testImplementation 'junit:junit:4.13.2'
+        testImplementation 'org.mockito:mockito-inline:5.1.0'
 
-        // https://developer.android.com/jetpack/androidx/releases/test/#1.2.0
-        api 'androidx.test:runner:1.2.0'
-        api 'androidx.test:rules:1.2.0'
-        api 'androidx.test.espresso:espresso-core:3.2.0'
+        api 'androidx.test:runner:1.+'
+        api 'androidx.test:rules:1.+'
+        api 'androidx.test.espresso:espresso-core:3.+'
```

Comment:
These could (assuming this works the way I assume, and a more specific dependency higher up that's compatible would just resolve to that) resolve to versions older than what we had before; is that okay?

E.g., if an app has `runner:1.1` like a bunch of flutter/packages, will this cause those apps to start trying to compile with 1.1 instead of optimistically upgrading to 1.2, and then fail because we actually needed 1.2?

Reply:
I think it will be ok because we don't actually need 1.2 but let me try to rewrite this so that we dont have the opportunity to find out.